### DISCUSSION TITLE
case insensitive regex

### DISCUSSION THREAD

**Discussion by ayadim** [projectdiscovery/nuclei#2781] [Q&A] [answered]
Hi 

 I want to know if I could ignore case sensitivity in my regular expression to match both "UpperCase and Lowercase" ? 
I hoped to find option like this : case-insensitive: true
Regards

**Comment by ayadim** [comment, marked as answer]
I found a solution add "(?i)" in the beginning of your regular expression.
Example : "(?i)key"
Example 2 : "(?i)(key|test)" 

I will leave this post for any one have the same issue.
regards

**Reply by ehsandeep** [reply]
`case-insensitive: true` is for the **word** matcher, for regex you can directly write case insensitive regex like you shared.
